src/ilasm/main.cpp

```diff
@@ -188,18 +188,18 @@ extern "C" int _cdecl wmain(int argc, __in WCHAR **argv)
       printf("\n/HIGHENTROPYVA  Set High Entropy Virtual Address capable PE32+ images (default for /APPCONTAINER)");
       printf("\n/NOCORSTUB      Suppress generation of CORExeMain stub");
       printf("\n/STRIPRELOC     Indicate that no base relocations are needed");
-      printf("\n/ITANIUM        Target processor: Intel Itanium");
       printf("\n/X64            Target processor: 64bit AMD processor");
```

Comment:
Nit: It makes sense to replace "64bit AMD" with just "x64".

Reply:
@AntonLapounov I agree this makes sense, but I want to scope this change to only ARM64 work, and not add general improvement/cleanup.